pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -41,6 +41,8 @@ const (
 )
 
 type kubenetNetworkPlugin struct {
+	network.NoopNetworkPlugin
```

Comment:
Please ignore my comment above. The shaper in kubenet is not complete. Ingress/egress for pod has not been covered.